src/main/java/io/vertx/openapi/contract/impl/OpenAPIContractImpl.java

```diff
@@ -201,4 +206,11 @@ public Operation findOperation(String urlPath, HttpMethod method) {
   public List<SecurityRequirement> getSecurityRequirements() {
     return securityRequirements;
   }
+
+  @Override
+  public SecurityScheme findSecurityScheme(String name) {
+    return securityRequirements != null ?
```

Comment:
`securityRequirements` is never null, and even if it is empty, a securityRequirements could be defined in the operation directly, couldn't it?